kmir/k-src/mir-lexer-syntax.md

```diff
@@ -20,7 +20,7 @@ This module defined the necessary `token` productions.
 ```k
   syntax IdentifierToken ::= r"[_a-zA-Z][_a-zA-Z0-9]*" [token]
   syntax LocalToken      ::= r"_[0-9]+"  [token]
-  syntax BBToken         ::= r"bb[0-9]+" [token]
+  syntax BBId         ::= r"bb[0-9]+" [token]
```

Comment:
I think that this overlaps with `IdentifierToken`. If that's the case, then it may happen that BBIds get parsed as IdentifierTokens, so you should probably use `prec` to give a higher priority to `BBId`. I think that the same applies to `LocalToken`.

Reply:
That's actually an interesting point. I think I will investigate how rustc handles that conflict and write a test for it. For now I will add `[prefer]` to `BBId`